mbed/CMakeLists.txt

```
@@ -0,0 +1,45 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+
+set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/mbed-os CACHE INTERNAL "")
+set(MBED_CONFIG_PATH ${CMAKE_CURRENT_BINARY_DIR} CACHE INTERNAL "")
+set(APP_TARGET BLE_PeriodicAdvertising)
```

`BLE_PeriodicAdvertising` seems out of place.

```suggestion
set(APP_TARGET mbed-bluetooth-power-consumption-benchmark)
```